verion 1.0.6 error "* txml/txml in ./node_modules/ol/node_modules/geotiff/src/geotiffimage.js"

system windows
install geotiff@1.0.6 error "* txml/txml in ./node_modules/ol/node_modules/geotiff/src/geotiffimage.js"

@constantinius unfortunatelly not, the message: 

> Module not found: Can't resolve 'txml/txml' in '/app/node_modules/geotiff /src'

is all i get from the dev server. not sure but in my case it seems like import has extra space after the `geotiff`

ERROR in [REDACTED]/node_modules/geotiff/src/geotiffimage.js
[REDACTED]: Module not found: Error: Can't resolve 'txml/txml' in '[REDACTED]/node_modules/geotiff/src'
[REDACTED]:  @ [REDACTED]/node_modules/geotiff/src/geotiffimage.js 4:0-34 778:19-24
[REDACTED]:  @ [REDACTED]/node_modules/geotiff/src/geotiff.js
[REDACTED]:  @ [REDACTED]/node_modules/ol/source/GeoTIFF.js
[REDACTED]:  @ [REDACTED]/node_modules/ol/source.js

--- a/package.json
+++ b/package.json
@@ -9,9 +9,6 @@
     "image",
     "raster"
   ],
-  "alias": {
-    "txml/txml": "txml/dist/txml"
-  },
   "main": "dist-node/geotiff.js",
   "module": "src/geotiff.js",
   "jsdelivr": "dist-browser/geotiff.js",
@@ -79,7 +76,8 @@
     "fs": false,
     "http": false,
     "https": false,
-    "url": false
+    "url": false,
+    "txml": "./node_modules/txml/dist/txml"
   },
   "sideEffects": false,
   "contributors": [

--- a/package.json
+++ b/package.json
@@ -77,7 +77,7 @@
     "http": false,
     "https": false,
     "url": false,
-    "txml": "./node_modules/txml/dist/txml"
+    "./src/txml": "./src/browser/txml"
   },
   "sideEffects": false,
   "contributors": [

--- a/src/browser/txml.js
+++ b/src/browser/txml.js
@@ -1,0 +1,1 @@
+export { parse } from 'txml/dist/txml';

--- a/src/geotiffimage.js
+++ b/src/geotiffimage.js
@@ -1,7 +1,7 @@
 /* eslint max-len: ["error", { "code": 120 }] */
 
 import { getFloat16 } from '@petamoriken/float16';
-import { parse } from 'txml/txml';
+import { parse } from 'txml';
 
 import { photometricInterpretations, ExtraSamplesValues } from './globals';
 import { fromWhiteIsZero, fromBlackIsZero, fromPalette, fromCMYK, fromYCbCr, fromCIELab } from './rgb';

--- a/src/geotiffimage.js
+++ b/src/geotiffimage.js
@@ -1,7 +1,7 @@
 /* eslint max-len: ["error", { "code": 120 }] */
 
 import { getFloat16 } from '@petamoriken/float16';
-import { parse } from 'txml';
+import { parse } from './txml';
 
 import { photometricInterpretations, ExtraSamplesValues } from './globals';
 import { fromWhiteIsZero, fromBlackIsZero, fromPalette, fromCMYK, fromYCbCr, fromCIELab } from './rgb';

--- a/src/txml.js
+++ b/src/txml.js
@@ -1,0 +1,1 @@
+export { parse } from 'txml';

